src/dpicker.js

```diff
@@ -178,9 +178,12 @@ function DPicker(element, options = {}) {
 
   let input = element.querySelector('input')
 
+  input.addEventListener('focus', input.blur)
   input.addEventListener('blur', this._events.inputBlur)
-  document.addEventListener('touchend', function() {
+  
+  document.addEventListener('touchend', event => {
     input.blur()
+    this._events.inputBlur(event)
   })
```

Comment:
Hmm, `input.blur()` should already call `this._events.inputBlur`

`focus` on the input can not call `blur`, it's its opposite, and it is added [here](https://github.com/soyuka/dpicker/blob/master/src/dpicker.js#L566). Blur was put there to fix some flickering issues (see https://github.com/soyuka/dpicker/issues/17)